tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature

```diff
@@ -0,0 +1,170 @@
+@asp
+@api
+@javascript
+
+Feature: Feature: test an Affinity Groups page
+  An Affinity Group page should display the title, logo, description, and tags.
+  Groups that have specified an optional Slack, Q&A, GitHub, or email link should
+  display a button with the appropriate link.
+  This test checks the following:
+    Join is & Leave an
+    Join on Slack (if applicable, links to slack group)
+    Visit Q&A Platform (if applicable, links to Ask.CI)
+    Mailing List (if applicable, links to email)
+    Github Organization (e.g. /affinity-groups/open-ondemand)
+    Coordinators (links to individual peoples profiles)
+    Events (if none, filler text “No upcoming events”)
+    Associated Resources accordion shows information about the resource on ACCESS RP AGs
+    Recommended Resources accordion shows recommendations on Community AGs
+
+  # TODO get this to work
+  # I have not be able to add an existing CI Link to the affinity group.
+  # The UI does not allow me to add an existing CI Link to the AG.
+  # The name of the CI Link gets appended with a variable suffix (like "ci-link-for-user-200 (5)")
+  # but I'm haven't figured out what that number should be ahead of time.
+  #Scenario: Admin user adds a CI Link to the AG ACCESS Support
+    Given I am logged in as a user with the "administrator" role
+    # add a CI-Link to an AG
+    When I go to "node/327/edit"
+    Then I should see "Edit Affinity Group ACCESS Support"
+    # this CI link is created by amp_dev.install
+    When I fill in "Display CI Links on your Affinity Group" with "ci-link-for-user-200"
+    And I wait 1 seconds
+    When I press "Save"
+    And I wait 2 seconds
+    Then I should see "Affinity Group ACCESS Support has been updated."
+    When I follow "ci-link-for-user-200"
+    And I wait 1 seconds
+    Then I should see "ci-link-for-user-200"
+
```

Comment:
Same as with the templates version of this test, I was not able to add a CI-Link to an AG.